### PR TITLE
project name validation

### PR DESCRIPTION
**Pull request by alexbruy**
when creating a new project check that project name does not contain invalid characters

### CODE REVIEW

File: Mergin/test/test_utils.py [hunk 1]
```diff
@@ -90,6 +90,81 @@ def test_datum_shift_grids(self):
...
+            ("projAUXect", True),
+            ("CONproject", True),
+            ("projectCON", True),
+            ("support", False),
+            ("helpdesk", False),
+            ("input", False),
+            ("lutraconsulting", False),
+            ("lutra", False),
+            ("merginmaps", False),
+            ("mergin", False),
```
Comment (varmar05) [inline Mergin/test/test_utils.py:162]: what about `admin` like? and maybe forbid `sales`?

Comment (alexbruy) [reply Mergin/test/test_utils.py:162]: Added both

File: Mergin/test/test_utils.py [hunk 2]
```diff
@@ -90,6 +90,83 @@ def test_datum_shift_grids(self):
...
+            ("admin", False),
+            ("sales", False),
```
Comment (tomasMizera) [inline Mergin/test/test_utils.py:164]: Can you please add the same Chinese/Hebrew project names as you did for mobile https://github.com/MerginMaps/input/pull/2449/files#diff-33c6f9baefe15bf8b0b47c49256319fad26445333a6bf988a5f797ef60dae96eR232-R233 ? thanks!

**Review by tomasMizera — APPROVED** [review]
Code is looking good! :)

**Comment by varmar05** [comment]
@tomasMizera I am wondering if we should not move this check to py-client as it would be useful for CLI as well
https://github.com/MerginMaps/mergin-py-client/blob/master/mergin/cli.py#L193

**Pull request merged**